javascriptv3/example_code/cross-services/textract-react/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "textract",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@aws-sdk/client-cloudformation": "^3.13.0",
+    "@aws-sdk/client-cognito-identity": "^3.9.0",
+    "@aws-sdk/client-cognito-identity-provider": "^3.9.0",
+    "@aws-sdk/client-s3": "^3.9.0",
+    "@aws-sdk/client-sqs": "^3.9.0",
+    "@aws-sdk/client-textract": "^3.9.0",
+    "@aws-sdk/credential-provider-cognito-identity": "^3.9.0",
+    "bootstrap": "^5.0.0-beta3",
```

Comment:
optional: use [react-bootstrap](https://react-bootstrap.github.io/)

It will reduce the amount of boilerplate code which needs to be written.
Example usage in notes-app https://github.com/aws-samples/aws-sdk-js-notes-app/search?q=react-bootstrap

Reply:
I agree react-bootstrap looks like the right choice, but the example is working as is and it looks like a fair effort to change over so I'm going to leave this one as is. Next time I will check out react-bootstrap!